src/language-server/config/loadConfig.ts

```diff
@@ -72,7 +72,7 @@ export async function loadConfig({
   requireConfig = false,
   name,
   type
-}: LoadConfigSettings) {
+}: LoadConfigSettings): Promise<ApolloConfig | undefined> {
```

Comment:
how about returning `ApolloConfig | null ` instead of `undefined` so that it's easier to tell that an empty value intentionally being returned vs an uninitialized var

Reply:
Yep 💯